src/Messages/Attachments/TSAttachmentStream.h

```diff
@@ -16,8 +17,6 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithContentType:(NSString *)contentType NS_DESIGNATED_INITIALIZER;
 - (instancetype)initWithPointer:(TSAttachmentPointer *)pointer NS_DESIGNATED_INITIALIZER;
 
-@property (atomic, readwrite) BOOL isDownloaded;
```

Comment:
I've promoted this property to the base class `TSAttachment` so that we can use it to determine if we need to apply the schema upgrade.